src/features/chains/ChainSelectModal.tsx

```diff
@@ -41,54 +43,3 @@ export function ChainSelectListModal({
     </Modal>
   );
 }
-
-// TODO update to support dynamic chain lists
```

Comment:
I was keeping this cause I suspected it would be useful later but it ended up confusing a junior dev who was looking through code, so just cutting